core/math/src/main/java/org/arakhne/afc/math/geometry/d3/ai/PathShadow3ai.java

```diff
@@ -282,8 +282,6 @@ private static void computeCrossings2(
 		final int shadowXmax = Math.max(shadow_x0, shadow_x1);
 		final int shadowYmin = Math.min(shadow_y0, shadow_y1);
 		final int shadowYmax = Math.max(shadow_y0, shadow_y1);
-		final int shadowZmin = Math.min(shadow_z0, shadow_z1);
```

Comment:
This change should be discarded because the 3D primitives are currently under intégration (see #47).